tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+  "include": ["lib/"],
```

Comment:
What is this for?

Reply:
Explicitly include which ts file to compile. By default tsconfig.json will search all the ts files within the root directory to compile, excluding those specified by "exclude". 

> https://www.typescriptlang.org/tsconfig#include

Reply:
Gotcha. I don't see any harm either way, so let's just leave it as-is.